operator-kustomize/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: operator
+  labels:
+    name: minio-operator
+  namespace: minio-operator
+spec:
+  type: LoadBalancer
```

Comment:
We don’t need type load balancer. Just clusterIP

Reply:
I used this based on @dvaldivia suggestion, do not have a preference.

Reply:
Loadbalancer is useful to expose the operator externally, also, there is only one instance of operator. will @dvaldivia chime in.

Reply:
Changed it to ClusterIP